### PR TITLE
Fix wording on Firefox sync

### PR DESCRIPTION
**Pull request by mfwmyfacewhen**
Resolves: #1196 

"it should be safe to use" makes it sound like we're unsure about recommending it. Removed this part so it is shorter and more confident.

### CODE REVIEW

File: docs/browsers.en.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ This prevents you from unintentionally connecting to a website in plain-text HTT
 
 #### Sync
 
-[Firefox Sync](https://hacks.mozilla.org/2018/11/firefox-sync-privacy/) allows your browsing data (history, bookmarks, etc.) to be accessible on all your devices. It is fully E2EE, so it should be safe to use.
+[Firefox Sync](https://hacks.mozilla.org/2018/11/firefox-sync-privacy/) allows your browsing data (history, bookmarks, etc.) to be accessible on all your devices. It is fully E2EE.
```
Comment (subatomic-honda-civic) [inline docs/browsers.en.md:101]: "It is fully E2EE." to me, sounds a bit *weird*, maybe somehow "integrate" that into the previous sentence? 

Something like

> ... to be accessible on all your devices while using E2EE.

**Review by subatomic-honda-civic — CHANGES_REQUESTED** [review]

File: docs/browsers.en.md (hunk 1 above)
Comment (mfwmyfacewhen) [reply docs/browsers.en.md:101]: Yeah I agree, maybe "Firefox Sync allows your browsing data (history, bookmarks, etc.) to be accessible on all your devices and protects it with E2EE."

**Review by subatomic-honda-civic — APPROVED** [review]
LGTM, Honda Civic Approved :+1:

**Review by freddy-m — APPROVED** [review]
LGTM

**Pull request merged**